### DISCUSSION TITLE
no anonymous write access error in intellij

### DISCUSSION THREAD

**Discussion by MaHi-312** [learndevopsonline/learndevopsonline#1520] [Q&A]
hi sir. i got remote .no anonymous write access error while pushing the frontend,backend code in intellij. i didnt get any useful information from google to solve that. could you please help me to solve this problem.

**Comment by sreedharm07** [comment]
can you please post screen shop if possible

**Reply by MaHi-312** [reply]
![Screenshot (17)](https://github.com/learndevopsonline/learndevopsonline/assets/142774205/1c4a26be-adb9-4105-a371-5dce0b7abffa)

**Reply by sreedharm07** [reply]
This repository was not there in your git repos.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/64387f5a-6e61-4aef-8461-4e56391c0742)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/77a94823-d3f7-4281-bbce-b00ef1b114f7)

**Reply by MaHi-312** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/142774205/480afef9-7b9d-410f-803b-bb23c209207a)

**Reply by MaHi-312** [reply]
I imported that backend and frontend repos from expenseapp-v1
